results/customer-portal/installer-platform.detailed-description.html

```diff
@@ -0,0 +1,12 @@
+The Red Hat JBoss Developer Studio Platform Installer scans for the products listed below using your PATH environment variable to determine if they're already installed. If not found, or an older version is installed, the following supported versions will then be installed for you by the installer:
```

Comment:
Do we really need to describe how this things works  ? it is an installer - it installs stuff.

i.e.
"Red Hat JBoss Developer Studio provide a single install of Red Hat JBoss Developer Studio, OpenJDK and all the tools needed to get started with container based development on Red Hat Container Development Kit 2.0.

<list of installed components, without versions and for Vagrant/VirtualBox add a (Optionally downloaded during install)"